dark-mode.css

```diff
@@ -139,12 +175,24 @@ body.dark-mode .sidebar-item > a {
 	color: inherit;
 }
 
-body.dark-mode .emoji-picker {
-	background-color: var(--rc-color-primary-dark);
+body.dark-mode input {
+	color: var(--color-white);
```

Comment:
this should probably be var(--color-dark) to solve the issue reported above

Reply:
err actually that breaks things in the front end. Like the channel search box. hrm.